CODE_OF_CONDUCT.md

```diff
@@ -31,6 +31,50 @@ Examples of unacceptable behavior by participants include:
   professional setting
 
+### Guide to Maintaining These Standards
+
+When interacting in our community, strive to maintain a friendly, empathetic tone. This includes, but is not limited to, interactions on Discord, GitHub, and Twitter.
+
+"Tone" refers to word, punctuation, and conversational choices that determine how your message _feels_ to others.
+
+Compare these two responses:
+
+_Question:_ Hey, I'm beginning with Astro. I tried to use `document` in my Astro page, but it says that the object doesn't exist. How can I manipulate the DOM?
+
+| Answer 1 | Answer 2 |
+| -- | -- |
+| You can't manipulate the DOM from an Astro file. Astro is a static site builder. | Hey, welcome to the community! <br/> Astro pages and components run on the server. Because of that, you won't have access to everything you're used to when writing JavaScript for the web. You're on the right track, though - you can access `document` from a `<script>` tag in your Astro file. <br/> Check out the docs here, and let me know if you need any more help! https://docs.astro.build/en/core-concepts/astro-components/#client-side-scripts |
+
+Answer 2 took more effort, but it made a significant difference. This response:
+
+- welcomed the new member into the community
+- made them feel validated for asking the question ("You're on the right track")
+- provided an explanation that fit the person who asked the question: _Answer 2_ sought to explain things to a self-described beginner, who might not understand the implications of "static site builder"
+- made it clear that the community is willing to help and that asking is not a burden ("let me know if you need any more help!")
+
+Our community prides itself on being _positive_, _helpful_, and _inclusive_. If you have these goals in mind when interacting in the community, you're on the right track!
+
+- To stay positive:
+
+  - Make it clear that you appreciate the other person's contributions.
+  - Aim to learn: even complaints can be valuable feedback.
+  - Look for alternate interpretations of your message before sending, and consider clarifying your intent if you're unsure how your words might come across (using exclamation points and emojis to convey friendliness is generally 👍 !)
+  - During conflict: maintain respect, avoid aggression, and avoid involving personal personal pronouns ("you").
+  - Set boundaries. Feel free to step away from a conversation when you feel frustrated.
+
+- To stay helpful:
+
+  - Seek to understand the the other person's desired outcomes. Are they looking for assistance? Just wanting to vent?
+  - When explaining something, first establish the knowledge level of the person you're responding to. Match your explanation to that level.
+  - Ask clarifying questions. If you're not sure what the person meant, ask them if your understanding is correct. If you're not sure what they already know, ask them if they're aware of a concept.
```

Comment:
```suggestion
  - Ask clarifying questions. If you're not sure what the person meant, ask them if your understanding is correct. If you're not sure what they already know, ask them if they're familiar with a concept.
```

^ just in case someone takes this literally and asks "are you aware of...?"